.woodpecker/docker.yml

```diff
@@ -1,3 +1,7 @@
+when:
+  event: [push, pull_request]
+  branch: [master, release/*]
```

Comment:
I would still prefer the following syntax:

(will try if it actually works in a test repo)

```yml
when:
  - event: pull_request
  - event: push
    branch: [master, release/*]
```